rules/rules-reviewed/quarkus/java-ee/cdi-to-quarkus.windup.xml

```diff
@@ -64,5 +64,34 @@
                 </iteration>
             </perform>
         </rule>
+        <rule id="cdi-to-quarkus-00040">
+            <when>
+                <javaclass references="javax.enterprise.inject.Produces">
```

Comment:
Nothing related to this line of code: I think we should do a rule also for [this commit](https://github.com/jboss-developer/jboss-eap-quickstarts/commit/8e24536ed72c93f5d3e5df844d84bd4978d5a7ad), in which the class annotated with `javax.ws.rs.ApplicationPath` and extending `javax.ws.rs.core.Application` is used to activate JAX-RS. The class is removed altogether.